playground/main.ts

```diff
@@ -1,24 +1,48 @@
-/* global OktaSignIn */
 /* eslint no-console: 0 */
 
-import signinWidgetOptions from '../.widgetrc.js';
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const signinWidgetOptions = require('../.widgetrc.js'); // commonJS module
```

Comment:
everyone would need to change their widgetrc to do `export default` instead of `module.exports`. Babel doesn't mind either way, but TS is pickier about wanting a "default" export for ES modules.

Reply:
I wonder if we can make this break change at next major release or so. just for consistency (all ESM)